pygsti/data/dataset.py

```diff
@@ -1619,6 +1619,16 @@ def add_cirq_trial_result(self, circuit, trial_result, key):
         key : str
             The string key of the measurement. Set by cirq.measure.
 
+        convert_int_to_binary : bool, optional (defaut True)
+            By default the keys in the cirq Results object are the integers representing
+            the bitstrings of the measurements on a set of qubits, in big-endian convention.
+            If True this uses the cirq function `cirq.big_endian_int_to_bits` to convert back
```

Comment:
I was wondering if we were going to get bit by endian-ness... Nice job covering that edge case!